### ISSUE TITLE
source = .user returns SSErrorDomain Code=100 "Cannot connect to iTunes Store"

### ISSUE THREAD

**Issue by fotiDim** [Alexsays/ASAppleMusic#1]
Steps to reproduce:
- Use the demo project as is
- Add `ASAppleMusic.shared.source = .user` right after initialization
- Try trough an iPhone device to make a search request. I used "us", "songs" and "binge" as term.

I am getting:
```
2018-06-28 23:11:26.629494+0200 ASAppleMusic_Example[334:14718] [core] "Error returned from daemon: Error Domain=com.apple.accounts Code=9 "(null)""
2018-06-28 23:11:26.629774+0200 ASAppleMusic_Example[334:14718] SSAccountStore: Failed to fetch the backing accounts. error = Error Domain=com.apple.accounts Code=9 "(null)"
2018-06-28 23:11:26.632496+0200 ASAppleMusic_Example[334:14810] [core] "Error returned from daemon: Error Domain=com.apple.accounts Code=9 "(null)""
2018-06-28 23:11:26.632698+0200 ASAppleMusic_Example[334:14810] SSAccountStore: Failed to fetch the backing accounts. error = Error Domain=com.apple.accounts Code=9 "(null)"
2018-06-28 23:11:26.644030+0200 ASAppleMusic_Example[334:14810] SSAccountStore: Unable to get the local account. error = Error Domain=SSErrorDomain Code=100 "Cannot connect to iTunes Store" UserInfo={NSLocalizedDescription=Cannot connect to iTunes Store}
[ASAppleMusic] 🛑: Missing token
```

**Comment by Alexsilvacodes** [comment]
Hi! Did you configured ASAppleMusic initially with your developer data? You need to call `ASAppleMusic.shared.initialize(keyID: String, teamID: String, tokenServer: String)` method. Let me know :)

**Comment by fotiDim** [comment]
@Alexsays yes of course I did. I kind of narrowed down the issue. If my developer token is expired then I need first make a call using `source = .developer` which fetches a new developer token. Only then I am able to switch to `source = .user` and make successful requests.

**Comment by fotiDim** [comment]
On further inspection I think the problem is how you pass the headers. According to the [documentation](https://developer.apple.com/documentation/applemusicapi/getting_keys_and_creating_tokens) it should be like 
```
curl -v -H 'Music-User-Token: [music user token]' -H 'Authorization: Bearer [developer token]' "https://api.music.apple.com/v1/catalog/us/songs/203709340"
```
but when I inspect the request from Alamofire it is the equivalent of
```
curl -v -H 'Authorization: Bearer [music user token]' "https://api.music.apple.com/v1/catalog/us/songs/203709340"
```

**Comment by fotiDim** [comment]
@Alexsays I am fixing the issue with this [PR](https://github.com/Alexsays/ASAppleMusic/pull/3)

**Comment by Alexsilvacodes** [comment]
Much appreciated, I will take a look. Thank you for your time :)

**Comment by Alexsilvacodes** [comment]
PR approved. I will close this issue

**Comment by Alexsilvacodes** [comment]
Pod released with your change in version 1.1.0.

Thank you another time
